docs/index.md

```diff
@@ -1,5 +1,32 @@
 # Dsp-Ingest
 
-The Dsp-Ingest service is designed to streamline and optimize the process of ingesting and managing data within the DaSCH Service Platform.
+The Dsp-Ingest service is designed to streamline and optimize the process of ingesting and managing data within the
+DaSCH Service Platform.
 
-It provides a RESTful [API](api-endpoints-projects.md) for the digital asset management.
\ No newline at end of file
+It provides a RESTful [API](api-endpoints-projects.md) for the digital asset management.
+
+# Assets
+
+Assets are the core of the Dsp-Ingest service. They can be images, videos, audio files, Excel file, etc.
```

Comment:
```suggestion
Assets are the core of the Dsp-Ingest service. They can be different sort of files, like: images, videos, audios, etc.
```

Reply:
Or this sentence might be removed, because it's kind of repeated in Type of assets paragraph